test/metrics/__snapshots__/regression.test.ts.snap

```diff
@@ -1,15 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`metrics:mean_absolute_error should reject invalid inputs 1`] = `
-Object {
-  "message": [Error: values passed to tensor(values) must be a number/boolean/string or an array of numbers/booleans/strings, or a TypedArray],
-  "name": "",
-}
-`;
+exports[`metrics:mean_absolute_error should reject invalid inputs 1`] = `[Error: values passed to tensor(values) must be a number/boolean/string or an array of numbers/booleans/strings, or a TypedArray]`;
 
 exports[`metrics:mean_absolute_error should reject invalid inputs 2`] = `[Error: Argument 'a' passed to 'sub' must be numeric tensor, but got string tensor]`;
 
-exports[`metrics:mean_absolute_error should reject invalid inputs 3`] = `[TypeError: The shapes of y_true [3] and y_pred [2] should be equal]`;
+exports[`metrics:mean_absolute_error should reject invalid inputs 3`] = `[TypeError: Shapes of y_true [3] and y_pred [2] should be equal]`;
```

Comment:
Once you update the error test to be based on matching the custom error type `ValidationError`, these snapshots should go away.

Reply:
Done